addon/components/initials-avatar.js

```diff
@@ -7,7 +7,21 @@ export default Ember.Component.extend({
   colorIndex: 1,
   maxColorIndex: 1,
 
-  hasImage: Ember.computed.notEmpty('image'),
+  checkImage: function() {
+    var self = this;
+    $.ajax(self.get('image'), {
+      success: function(resp) {
+        self.set('hasImage', true);
+      },
+    });
```

Comment:
It seems like we could put the hasImage back to its original definition and have a controller somewhere else do the ajax request and assign the image property here if it exists. Initially it would just be set to null.